src/context.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from 'vscode';
+import { SwiftPackage } from './package';
+
+export class SwiftContext {
+	private constructor(
+        public workspaceRoot: string,
+        public extContext: vscode.ExtensionContext,
+        public spmPackage: SwiftPackage
```

Comment:
This should probably also be renamed? In this case, it can just be `package` I think?

Reply:
Can't rename it to `package` as it is a reserved word. Changed to `swiftPackage`